### PR TITLE
fixup setup to run first time and not fail if qm.service is not enabled

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@lsm5 @alexlarsson PTAL

File: setup [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -ex
+#! /bin/sh -ex
```
Comment (lsm5) [inline setup:1]: just double checking this was intentional.

File: setup [hunk 2]
```diff
@@ -15,16 +15,13 @@ INSTALLDIR="$1"
 ROOTFS="$2"
 [ ! -z "${ROOTFS}" ] || ROOTFS=/usr/lib/qm/rootfs
 
-. /etc/os-release
-
-systemctl stop qm 2> /dev/null
+systemctl stop qm.service 2>/dev/null || true
 
 install() {
     rootfs=$1
```
Comment (lsm5) [inline setup:21]: shouldn't this be $2

**Review by lsm5 — COMMENTED** [review]
LGTM mostly, except a couple of issues

File: setup (hunk 2 above)
Comment (rhatdan) [reply setup:21]: No it is $2 for the script but this is passed in as $1 in the function.

**Pull request merged**